src/t2iapi/operation/service.proto

```diff
@@ -29,4 +30,11 @@ service OperationService {
     rpc SetOperatingMode (t2iapi.operation.SetOperatingModeRequest)
         returns (BasicResponse);
 
+    /*
+    Update the @InvocationEffectiveTimeout of the pm:AbstractOperationDescriptor with the given handle.
```

Comment:
use the same wording. Update should be Modify in the description of the rpc.

Reply:
done

Reply:
after discussion i also added missing "an SDC operation invocation"